Wikipedia/UI-V5/WMFArticleListDataSource.h

```diff
@@ -17,9 +18,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (MWKHistoryDiscoveryMethod)discoveryMethod;
 
+- (void)setSavedPageList:(MWKSavedPageList*)savedPageList;
```

Comment:
Any reason for this to be mutable as opposed to initing with it?

Also, besides setting this property, the Article List VC really doesn't need to know that a data source has a saved page list, right? All the cells currently have a save button, but the Article VC doesn't know that. Maybe this points to this not really being the right place for this.